apps/extension/public/manifest.json

```diff
@@ -16,12 +16,12 @@
   },
   "content_scripts": [
     {
-      "matches": ["https://*/*"],
+      "matches": ["https://*/*", "http://localhost/*"],
```

Comment:
this line, instead of the previously used one, must work correctly according to the docs: https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns#special

Reply:
Just checked a test upload, it works 👍